src/xpk/core/core.py

```diff
@@ -2623,6 +2624,11 @@ def get_volumes(args, system: SystemCharacteristics) -> str:
                   claimName: {storage.pvc}
                   readOnly: {storage.readonly}
               """
+    if storage.type == GCP_FILESTORE_TYPE:
+      volumes += f"""- name: {storage.pv}
+                persistentVolumeClaim:
+                  claimName: {storage.pvc}
```

Comment:
What about a readOnly flag? Does filestore supports it? If not we should probably validate it when adding storage.

Reply:
It is supported by filestore. Current behavior is that readonly can be defined in Persistent Volume yaml file and it will be applied in volume mount of this storage. If we use --readonly=true/false it will overwrite readonly value from PersistentVolume definiton.